rust_src/remacs-sys/README.md

```diff
@@ -0,0 +1,108 @@
+# remacs-sys
+
+`remacs-sys` is Rust library containing all [Foreign Function Interface (FFI)](https://doc.rust-lang.org/book/first-edition/ffi.html) declarations of structs and functions that are defined in C. Then, they can be used in the Rust code. C implementations are in `remacs/src`.
+
+## Example: Use `make_float` C function in Rust
+
+First, let's have a look how `make_float` is defined historically in the C code. Then, how one can re-use it in Rust.
+
+### On the C side
+
+`make_float` is regular C function . The declaration can be found in the header file `remacs/src/lisp.h`:
```

Comment:
*is a regular C function

I'd be tempted to say "is a C function" -- I'm not sure we gain anything by saying 'regular'.